simpleline/render/adv_widgets.py

```diff
@@ -176,3 +176,71 @@ def input(self, args, key):
 
     def prompt(self, args=None):
         return Prompt(_("Press %s to return") % Prompt.ENTER)
+
+
+class GetInputScreen(UIScreen):
+    """Screen for getting user input."""
+
+    def __init__(self, message):
+        """
+        :param message: Prompt printed before user input.
+        :type message: str
+        """
+        super().__init__()
+        self._message = message
+        self._value = None
+        self._conditions = []
+
+    @property
+    def value(self):
+        """User input."""
+        return self._value
+
+    def add_acceptance_condition(self, acceptance_function, args=None):
+        """Add acceptance condition to the conditions list.
+
+        :param acceptance_function: Functions that accepts or rejects a user input.
+        :type acceptance_function: `[function(input, error_report) -> bool]`  - array of functions which takes
```

Comment:
The `acceptance_function` is really an array of functions? Couldn't we add the functions one by one?

Reply:
Nope it is not. This is error in documentation.
Fixed.